--- ./configure.ac
+++ ./configure.ac
@@ -46,8 +46,8 @@
 
 AC_CHECK_LIB(m, log10)
 
-PKG_CHECK_MODULES(AVFORMAT, [libavformat >= 59.27.100])
-PKG_CHECK_MODULES(AVCODEC, [libavcodec >= 59.37.100])
-PKG_CHECK_MODULES(AVUTIL, [libavutil >= 57.28.100])
+PKG_CHECK_MODULES(AVFORMAT, [libavformat >= 59.16.100])
+PKG_CHECK_MODULES(AVCODEC, [libavcodec >= 59.18.100])
+PKG_CHECK_MODULES(AVUTIL, [libavutil >= 57.17.100])
 
 AM_OPTIONS_WXCONFIG
 reqwx=3.0.0
